ckb-vm-bench-scripts/contracts/ed25519/Makefile

```diff
@@ -35,15 +35,29 @@ endif
 default: build test
 
 build:
-	RUSTFLAGS="$(FULL_RUSTFLAGS)" TARGET_CC="$(CLANG)" TARGET_AR="$(AR)" \
-		cargo build --target=riscv64imac-unknown-none-elf $(MODE_ARGS) $(CARGO_ARGS)
+	for binary in $(BINARIES); do \
+		binary=$${binary}_ckbvm; \
+		RUSTFLAGS="$(FULL_RUSTFLAGS)" TARGET_CC="$(CLANG)" TARGET_AR="$(AR)" \
+			cargo build --target=riscv64imac-unknown-none-elf --bin $$binary $(MODE_ARGS) $(CARGO_ARGS); \
+	done
+	for binary in $(BINARIES); do \
```

Comment:
I'm just curious: why using 2 for loops? What if for each binary we first build the ckbvm target then the ckbvm target?